lib/model/migrations/20220829-01-create-entities-defs.js

```diff
@@ -0,0 +1,39 @@
+// Copyright 2022 ODK Central Developers
+// See the NOTICE file at the top-level directory of this distribution and at
+// https://github.com/getodk/central-backend/blob/master/NOTICE.
+// This file is part of ODK Central. It is subject to the license terms in
+// the LICENSE file found in the top-level directory of this distribution and at
+// https://www.apache.org/licenses/LICENSE-2.0. No part of ODK Central,
+// including this file, may be copied, modified, propagated, or distributed
+// except according to the terms contained in the LICENSE file.
+
+const up = (db) =>
+  db.schema.createTable('entities', (entities) => {
+    entities.increments('id').primary();
+    entities.string('uuid').notNull();
+    entities.integer('datasetId');
+    entities.text('label').notNull();
+    entities.dateTime('createdAt');
+    entities.integer('createdBy').notNull();
```

Comment:
I think we should add `updatedAt` as well, it will help us identify if dataset has been changed